independent-publisher-connectors/Fuzzy Search/readme.md

```diff
@@ -0,0 +1,18 @@
+## Fuzzy Search
```

Comment:
@happyume please include all required section in the readme file. please follow our template [here ](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/Independent%20Publisher/readme.md)and for raw code view [here](https://raw.githubusercontent.com/microsoft/PowerPlatformConnectors/dev/templates/Independent%20Publisher/readme.md).